README.md

```diff
@@ -1,5 +1,14 @@
 # WeakRefs TC39 proposal
 
+## Status
+
+* WeakRef and FinalizationRegistry are now Stage 4, since the July 2020 TC39 meeting
+* V8 -- shipping [Chrome 84](https://v8.dev/features/weak-references)
```

Comment:
It's under flag in Chrome 84 and is available by default in 85 (beta, stable release is 25 Aug)